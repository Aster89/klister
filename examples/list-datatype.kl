#lang "prelude.kl"

(import "list.kl")

(example (reverse (list 1 2 3)))
