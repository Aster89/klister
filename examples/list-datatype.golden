(:: Signal 3 (:: Signal 2 (:: Signal 1 (nil)))) : (List Signal)
